### ISSUE TITLE
global copy doesn't work

### ISSUE THREAD

**Issue by cldougl** [plotly/dash-table#75]
users cannot copy via the browser (chrome edit->copy)
note: pasting from the browser does work.

**Comment by andreapiso** [comment]
This issue should be reopened in my opinion. At the moment, copying from browser only seem to work when the table is editable. 

As an example, I tried to copy the content of the cell "New Mexico" on the simple table at https://dash.plot.ly/datatable/, and nothing gets copied in the clipboard.

**Comment by Marc-Andre-Rivet** [comment]
@andreapiso Thanks for finding this bug. I would suggest that you create a new issue with your findings instead in the future. Re-opening old GitHub issues makes the issue's history hard to follow. Also, the issue you are describing is related but not the same as the issue originally covered by here.

I will create an issue and link it to your comment above.

**Comment by andreapiso** [comment]
Thanks for creating the issue. Sorry for piggy-backing on the old thread, was afraid to open a duplicate.
